rotkehlchen/chain/evm/decoding/curve/decoder.py

```diff
@@ -559,6 +540,21 @@ def _handle_post_decoding(
                 )
                 break  # We only need to handle zap contracts once
 
+        for event in decoded_events:
+            if (
+                event.address == ZERO_ADDRESS and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.asset.is_evm_token() and
+                (event_asset := event.asset.resolve_to_evm_token()).evm_address in self.gauges
```

Comment:
have you checked the case for eth on this?

Reply:
We look for the gauge token here so it cannot be a native currency